components/base/sensorcontrolled/velocities.go

```diff
@@ -84,6 +95,18 @@ func (sb *sensorBase) setupControlLoop(linear, angular control.PIDConfig) error
 	return nil
 }
 
+func (sb *sensorBase) waitForTuning() {
+	for {
+		if !control.TunedPIDVals[0].NeedsAutoTuning() && !control.TunedPIDVals[1].NeedsAutoTuning() {
+			sb.tunedVals[0] = control.TunedPIDVals[0]
+			sb.tunedVals[1] = control.TunedPIDVals[1]
+			control.UpdateTunedPIDBlock(sb.controlLoopConfig, control.LinearPIDIndex, sb.tunedVals[0])
```

Comment:
wondering if theres another way to do `control.LinearPIDIndex`, but I don't think we will be able to until we have multi